spec/text_message_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'text_message'
+
+describe TextMessage do
+  subject(:text) { described_class.new(client) }
```

Comment:
This really helps improve test readability esp. when frequently switching between files